main.go

```diff
@@ -196,13 +196,13 @@ func main() {
 		&cli.StringFlag{
 			Name:    "s3.iam_role_endpoint",
 			Value:   "",
-			Usage:   "Endpoint for using IAM security credentials, eg http://169.254.169.254 for EC2, http://169.254.170.2 for ECS.",
+			Usage:   "Endpoint for using IAM security credentials. By default it will look for credentials in the standard locations for the AWS platform being ran on.",
```

Comment:
nitpick: let's just say "for the AWS platform."

Reply:
Sure :)